hlx_statics/blocks/side-nav/side-nav.js

```diff
@@ -2,54 +2,184 @@ import {
   applyAnalyticHeaderOverride,
   createTag
 } from '../../scripts/lib-adobeio.js';
-import {getMetadata } from '../../scripts/lib-helix.js';
-import { loadFragment } from '../fragment/fragment.js';
+import { getMetadata } from '../../scripts/lib-helix.js';
+// import { loadFragment } from '../fragment/fragment.js';
 
 /**
- * decorates the side-nav
+ * Decorates the side-nav
  * @param {Element} block The site-nav block element
  */
 export default async function decorate(block) {
-  const navigationLinks = createTag('nav', { role: 'navigation'});
+  const navigationLinks = createTag('nav', { role: 'navigation' });
   navigationLinks.setAttribute('aria-label', 'Primary');
 
   const navigationLinksContainer = createTag('div');
   navigationLinks.append(navigationLinksContainer);
 
-  const navigationLinksUl = createTag('ul', {role: 'tree', class: 'spectrum-SideNav spectrum-SideNav--multiLevel'});
+  const navigationLinksUl = createTag('ul', { role: 'tree', class: 'spectrum-SideNav spectrum-SideNav--multiLevel' });
   navigationLinksUl.setAttribute('aria-label', 'Table of contents');
   navigationLinksContainer.append(navigationLinksUl);
 
-  let pathPrefix = getMetadata('pathprefix').replace('/', '');;
+  let pathPrefix = getMetadata('pathprefix').replace('/', '');
   let navPath = `${window.location.origin}/${pathPrefix}/config`;
 
-  const fragment = await loadFragment(navPath);
```

Comment:
Hey @BaskarMitrah - so I wanted us to use the `loadFragment` for when we do requests to assets like the nav. I can make the changes back to using fragments since you're out.